### PR TITLE
[action] appetize add timeout support

### PR DESCRIPTION
**Pull request by raulriera**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context
Adding support for the `timeout` option in Appetize, as seen in their documentation https://docs.appetize.io/api/update-existing-app it would be ideal to support more than the default 2mins.

### Description
Expanded the `appetize` and `build_and_upload_to_appetize` actions to include this parameter. Also included the appropriate validations and test to make sure we always submit the values specified by Apptize themselves.

### Testing Steps
Running `bundle exec rspec ./fastlane/spec/actions_specs/appetize_spec.rb ` will result in green tests.

### CODE REVIEW

**Comment by raulriera** [comment]
@googlebot I signed it!

File: fastlane/lib/fastlane/actions/build_and_upload_to_appetize.rb [hunk 1]
```diff
@@ -21,7 +21,8 @@ def self.run(params)
         other_action.appetize(path: zipped_bundle,
                               api_token: params[:api_token],
                               public_key: params[:public_key],
-                              note: params[:note])
+                              note: params[:note],
+                              timeout: params[:timeout])
```
Comment (raulriera) [inline fastlane/lib/fastlane/actions/build_and_upload_to_appetize.rb:25]: Wasn't able to find a test for using this file

File: fastlane/swift/Fastlane.swift [hunk 2]
```diff
@@ -230,15 +230,17 @@ public func appetize(apiHost: String = "api.appetize.io",
                      platform: String = "ios",
                      path: String? = nil,
                      publicKey: String? = nil,
-                     note: String? = nil)
+                     note: String? = nil,
+                     timeout: Integer? = nil)
```
Comment (Caiopia) [inline fastlane/swift/Fastlane.swift:234]: Can't comment on the exact line, but we could update the docs for this new parameter too. Same with the `buildAndUploadToAppetize` function below.

File: fastlane/swift/Fastlane.swift [hunk 3]
```diff
@@ -882,13 +884,15 @@ public func buildAndUploadToAppetize(xcodebuild: [String: Any] = [:],
                                      scheme: String? = nil,
                                      apiToken: String,
                                      publicKey: String? = nil,
-                                     note: String? = nil)
+                                     note: String? = nil,
+                                     timeout: Integer?)
```
Comment (Caiopia) [inline fastlane/swift/Fastlane.swift:888]: Any reason this shouldn't have the same default value as above?

**Review by Caiopia — COMMENTED** [review]
Code looks good to me, just some minor suggestions. Haven't tested this yet though.

**Review by minuscorp — CHANGES_REQUESTED** [review]
Initially it seems good to me. A few notes, don't push Swift changes add they are autogenerated by fastlane, and add to the `ConfigItem` the `:type` key to pass it correctly to the Swift code (I.e. `Integer`)

**Comment by raulriera** [comment]
Hey @minuscorp , what are the request changes? GitHub is not showing any information

**Comment by raulriera** [comment]
Thanks for the review @minuscorp, code is updated. I didn't found anything in the docs about generating the swift code. Is there a step I need to do for this?

**Comment by minuscorp** [comment]
Just try to update it locally and see if the changes that you expect are done, there's a lane called `generate_swift_api` or similar that updates and build the whole interface and SPM package.

**Comment by raulriera** [comment]
Thanks, that did it. Should I include the output of all that generation here? (more files were changed) or is that only a release process?

**Comment by minuscorp** [comment]
That's part of the release process, but that way you can make sure that your changes are good. Useful for contributions 🎉

**Review by minuscorp — APPROVED** [review]
LGTM 🎉

**Review by joshdholtz — APPROVED** [review]
This is 🔥 Thanks so much for adding this! Really appreciate the contribution ❤️

**Pull request merged**